vercel.json

```diff
@@ -2,7 +2,7 @@
   "functions": {
     "api/*.js": {
       "memory": 128,
-      "maxDuration": 30
+      "maxDuration": 10
```

Comment:
You're probably aware of this and added this commit to test the pull request on your own vercel instance. I will add this here to be sure. This change should not be present in the final pull request see https://github.com/anuraghazra/github-readme-stats/issues/1416

Reply:
You can also test a pull request without deploying using the [vercel client](https://vercel.com/cli) (see https://github.com/vercel/vercel/issues/2864#issuecomment-963533339).

Reply:
The pull request is working on my localhost:

![image](https://user-images.githubusercontent.com/17570430/144722335-e90215ce-c7f1-4ee0-b4f0-f78daaf795d3.png)

To be really sure. I will manually double-check if there were no other conflicts that went undetected.